modules/installation-performing-disconnected-mirror.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc
+
+[id="installation-performing-disconnected-mirror"]
+= Performing a mirror to disk for use in disconnected environments
+
+When you mirror images to disk, you download images as files. Then, you move your
+mirror host, which can be a laptop or a removable media device, like a
+USB drive, into the restricted network and complete the mirror
+procedure.
+
+.Procedure
+
+. On a local host, test the mirror process:
++
+----
+$ oc adm release mirror <product_version> --to <mirror_repository> --dry-run
+----
+
+. On a local host, copy the ICS/ICSP:
```

Comment:
From user perspective, What is "ICS/ICSP"? How to consume it?

Reply:
[`imageContentSources` and `ImageContentSourcePolicy`][1].  We should probably spell them out and not abbreviate.

[1]: https://github.com/openshift/installer/blob/dd27e4cc4e2b433216dee8a0e9121c635a824de3/docs/dev/alternative_release_image_sources.md